### ISSUE TITLE
Accessibility - missing label / title in Grid controls

### ISSUE THREAD

**Issue by shekerev-progress** [telerik/kendo-react#381]
## Current behavior
Some grid controls are missing label and title attributes:
- input and select elements in filters in Grid
- select in Grid pager

## Expected behavior
[WCAG 21 Rule](https://www.w3.org/WAI/WCAG21/Techniques/html/H65) - The objective of this technique is to use the title attribute to label form controls when the visual design cannot accommodate the label (for example, if there is no text on the screen that can be identified as a label) or where it might be confusing to display a label. User agents, including assistive technology, can speak the title attribute.
Error - Control has no LABEL and no TITLE attribute.

## Minimal reproduction of the problem with instructions
Reported by client in [this demo](https://www.telerik.com/kendo-react-ui/components/grid/)
Tool client used is Accessibility Insights for Web

## QA
Agreed with dev to run a QA cycle on accessibility after this one.

## Side Note
On [the same demo](https://www.telerik.com/kendo-react-ui/components/grid/) our DropDownList component, used for intl provider, is also missing a title, but I am not so sure if this is a problem. Please advise

**Comment by melb47** [comment]
![image](https://user-images.githubusercontent.com/65355931/142650065-39a2314a-0e64-4dbe-a2a5-a958206d7b3e.png)

I am experiencing these same accessibility issues that I am seeing on the KendoReact page. The accessibility issues I circled in the screenshot are also showing up for my code. Is there an ETA for a fix?

**Comment by filipKovachev** [comment]
The ensure every form element has a label error is reported again in ticket: 1548103

**Comment by nstoychev** [comment]
Implemented in 5.10.0.
